views/layouts/main.handlebars

```diff
@@ -29,16 +29,21 @@
     <a href="https://qsurvey.mozilla.com/s3/txp-firefox-send" rel="noreferrer noopener" class="feedback" target="_blank" data-l10n-id="siteFeedback">Feedback</a>
   </header>
   <div class="all">
+    <noscript>
+      <h2>Firefox Send requires JavaScript</h2>
+      <p><a href="https://github.com/mozilla/send/blob/master/docs/faq.md#why-does-firefox-send-require-javascript" target="_blank" rel="noreferrer noopener">Why does Firefox Send require JavaScript?</a></p>
+      <p>Please enable JavaScript and try again.</p>
+    </noscript>
```

Comment:
Unless there are a11y concerns, I'm quite OK with this here being the suggested solution. Can we assume that people who disable JavaScript are tech-savvy enough to understand this message?